Tests are failing in OpenShift CI

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/13471/rehearse-13471-pull-ci-freeipa-freeipa-operator-master-unit/1329040777505935360
```
Running Suite: Controller Suite
===============================
Random Seed: 1605703216
Will run 0 of 0 specs
STEP: bootstrapping test environment
2020-11-18T12:40:16.800Z	DEBUG	controller-runtime.test-env	starting control plane	{"api server flags": []}
2020-11-18T12:40:16.805Z	ERROR	controller-runtime.test-env	unable to start the controlplane	{"tries": 0, "error": "fork/exec /usr/local/kubebuilder/bin/etcd: no such file or directory"}
github.com/go-logr/zapr.(*zapLogger).Error
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/go-logr/zapr/zapr.go:128
sigs.k8s.io/controller-runtime/pkg/envtest.(*Environment).startControlPlane
	/go/src/github.com/freeipa/freeipa-operator/vendor/sigs.k8s.io/controller-runtime/pkg/envtest/server.go:279
sigs.k8s.io/controller-runtime/pkg/envtest.(*Environment).Start
	/go/src/github.com/freeipa/freeipa-operator/vendor/sigs.k8s.io/controller-runtime/pkg/envtest/server.go:241
github.com/freeipa/freeipa-operator/controllers.glob..func1
	/go/src/github.com/freeipa/freeipa-operator/controllers/suite_test.go:61
reflect.Value.call
	/usr/local/go/src/reflect/value.go:475
reflect.Value.Call
	/usr/local/go/src/reflect/value.go:336
github.com/onsi/ginkgo/internal/leafnodes.newRunner.func1
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/onsi/ginkgo/internal/leafnodes/runner.go:49
github.com/onsi/ginkgo/internal/leafnodes.(*runner).runAsync.func1
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/onsi/ginkgo/internal/leafnodes/runner.go:86
2020-11-18T12:40:16.805Z	ERROR	controller-runtime.test-env	unable to start the controlplane	{"tries": 1, "error": "fork/exec /usr/local/kubebuilder/bin/etcd: no such file or directory"}
github.com/go-logr/zapr.(*zapLogger).Error
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/go-logr/zapr/zapr.go:128
sigs.k8s.io/controller-runtime/pkg/envtest.(*Environment).startControlPlane
	/go/src/github.com/freeipa/freeipa-operator/vendor/sigs.k8s.io/controller-runtime/pkg/envtest/server.go:279
sigs.k8s.io/controller-runtime/pkg/envtest.(*Environment).Start
	/go/src/github.com/freeipa/freeipa-operator/vendor/sigs.k8s.io/controller-runtime/pkg/envtest/server.go:241
github.com/freeipa/freeipa-operator/controllers.glob..func1
	/go/src/github.com/freeipa/freeipa-operator/controllers/suite_test.go:61
reflect.Value.call
	/usr/local/go/src/reflect/value.go:475
reflect.Value.Call
	/usr/local/go/src/reflect/value.go:336
github.com/onsi/ginkgo/internal/leafnodes.newRunner.func1
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/onsi/ginkgo/internal/leafnodes/runner.go:49
github.com/onsi/ginkgo/internal/leafnodes.(*runner).runAsync.func1
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/onsi/ginkgo/internal/leafnodes/runner.go:86
2020-11-18T12:40:16.805Z	ERROR	controller-runtime.test-env	unable to start the controlplane	{"tries": 2, "error": "fork/exec /usr/local/kubebuilder/bin/etcd: no such file or directory"}
github.com/go-logr/zapr.(*zapLogger).Error
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/go-logr/zapr/zapr.go:128
sigs.k8s.io/controller-runtime/pkg/envtest.(*Environment).startControlPlane
	/go/src/github.com/freeipa/freeipa-operator/vendor/sigs.k8s.io/controller-runtime/pkg/envtest/server.go:279
sigs.k8s.io/controller-runtime/pkg/envtest.(*Environment).Start
	/go/src/github.com/freeipa/freeipa-operator/vendor/sigs.k8s.io/controller-runtime/pkg/envtest/server.go:241
github.com/freeipa/freeipa-operator/controllers.glob..func1
	/go/src/github.com/freeipa/freeipa-operator/controllers/suite_test.go:61
reflect.Value.call
	/usr/local/go/src/reflect/value.go:475
reflect.Value.Call
	/usr/local/go/src/reflect/value.go:336
github.com/onsi/ginkgo/internal/leafnodes.newRunner.func1
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/onsi/ginkgo/internal/leafnodes/runner.go:49
github.com/onsi/ginkgo/internal/leafnodes.(*runner).runAsync.func1
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/onsi/ginkgo/internal/leafnodes/runner.go:86
2020-11-18T12:40:16.806Z	ERROR	controller-runtime.test-env	unable to start the controlplane	{"tries": 3, "error": "fork/exec /usr/local/kubebuilder/bin/etcd: no such file or directory"}
github.com/go-logr/zapr.(*zapLogger).Error
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/go-logr/zapr/zapr.go:128
sigs.k8s.io/controller-runtime/pkg/envtest.(*Environment).startControlPlane
	/go/src/github.com/freeipa/freeipa-operator/vendor/sigs.k8s.io/controller-runtime/pkg/envtest/server.go:279
sigs.k8s.io/controller-runtime/pkg/envtest.(*Environment).Start
	/go/src/github.com/freeipa/freeipa-operator/vendor/sigs.k8s.io/controller-runtime/pkg/envtest/server.go:241
github.com/freeipa/freeipa-operator/controllers.glob..func1
	/go/src/github.com/freeipa/freeipa-operator/controllers/suite_test.go:61
reflect.Value.call
	/usr/local/go/src/reflect/value.go:475
reflect.Value.Call
	/usr/local/go/src/reflect/value.go:336
github.com/onsi/ginkgo/internal/leafnodes.newRunner.func1
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/onsi/ginkgo/internal/leafnodes/runner.go:49
github.com/onsi/ginkgo/internal/leafnodes.(*runner).runAsync.func1
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/onsi/ginkgo/internal/leafnodes/runner.go:86
2020-11-18T12:40:16.807Z	ERROR	controller-runtime.test-env	unable to start the controlplane	{"tries": 4, "error": "fork/exec /usr/local/kubebuilder/bin/etcd: no such file or directory"}
github.com/go-logr/zapr.(*zapLogger).Error
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/go-logr/zapr/zapr.go:128
sigs.k8s.io/controller-runtime/pkg/envtest.(*Environment).startControlPlane
	/go/src/github.com/freeipa/freeipa-operator/vendor/sigs.k8s.io/controller-runtime/pkg/envtest/server.go:279
sigs.k8s.io/controller-runtime/pkg/envtest.(*Environment).Start
	/go/src/github.com/freeipa/freeipa-operator/vendor/sigs.k8s.io/controller-runtime/pkg/envtest/server.go:241
github.com/freeipa/freeipa-operator/controllers.glob..func1
	/go/src/github.com/freeipa/freeipa-operator/controllers/suite_test.go:61
reflect.Value.call
	/usr/local/go/src/reflect/value.go:475
reflect.Value.Call
	/usr/local/go/src/reflect/value.go:336
github.com/onsi/ginkgo/internal/leafnodes.newRunner.func1
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/onsi/ginkgo/internal/leafnodes/runner.go:49
github.com/onsi/ginkgo/internal/leafnodes.(*runner).runAsync.func1
	/go/src/github.com/freeipa/freeipa-operator/vendor/github.com/onsi/ginkgo/internal/leafnodes/runner.go:86
Failure [0.007 seconds]
[BeforeSuite] BeforeSuite 
/go/src/github.com/freeipa/freeipa-operator/controllers/suite_test.go:52
  Unexpected error:
      
      <*fmt.wrapError | 0xc0003e0c00>: {
          msg: "failed to start the controlplane. retried 5 times: fork/exec /usr/local/kubebuilder/bin/etcd: no such file or directory",
          err: {
              Op: "fork/exec",
              Path: "/usr/local/kubebuilder/bin/etcd",
              Err: 0x2,
          },
      }
      failed to start the controlplane. retried 5 times: fork/exec /usr/local/kubebuilder/bin/etcd: no such file or directory
  occurred
  /go/src/github.com/freeipa/freeipa-operator/controllers/suite_test.go:62
------------------------------
Ran 0 of 0 Specs in 0.007 seconds
FAIL! -- 0 Passed | 0 Failed | 0 Pending | 0 Skipped
--- FAIL: TestAPIs (0.01s)
FAIL
coverage: 0.0% of statements
FAIL	github.com/freeipa/freeipa-operator/controllers	0.028s
FAIL
make: *** [test] Error 1
2020/11/18 12:40:17 Container test in pod unit failed, exit code 2, reason Error
2020/11/18 12:40:17 No custom metadata found and prow metadata already exists. Not updating the metadata.
2020/11/18 12:40:17 Ran for 3m45s
error: some steps failed:
  * could not run steps: step unit failed: test "unit" failed: the pod ci-op-2zmbpxqx/unit failed after 1m3s (failed containers: test): ContainerFailed one or more containers exited
Container test exited with code 2, reason Error
```

Apparently `test` target in `Makefile` generated by operator-sdk doesn't have all the required steps, see  https://github.com/operator-framework/operator-sdk/issues/3692

--- a/.gitignore
+++ b/.gitignore
@@ -28,3 +28,6 @@ docker-image-freeipa-operator.tar.gz
 
 # Ignore local go directory
 /go
+
+# Ignore directory containing envtest binaries
+testbin/*

--- a/Makefile
+++ b/Makefile
@@ -52,8 +52,11 @@ all: manager
 FORCE:
 
 # Run tests
+ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 test: generate fmt vet manifests
-	go test ./... -coverprofile cover.out
+	mkdir -p "${ENVTEST_ASSETS_DIR}"
+	test -f "${ENVTEST_ASSETS_DIR}/setup-envtest.sh" || curl -sSLo "${ENVTEST_ASSETS_DIR}/setup-envtest.sh" https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh
+	source "${ENVTEST_ASSETS_DIR}/setup-envtest.sh"; fetch_envtest_tools "$(ENVTEST_ASSETS_DIR)"; setup_envtest_env "$(ENVTEST_ASSETS_DIR)"; go test ./... -coverprofile cover.out
 
 # Build manager binary
 manager: generate fmt vet

